### DISCUSSION TITLE
Divider with text?

### DISCUSSION THREAD

**Discussion by georgebrianb** [chakra-ui/chakra-ui#5700] [Q&A]
Hey guys, I just started out using Chakra UI and I was wondering if it's possible to have some text within the divider similar to this: 

![image](https://user-images.githubusercontent.com/32679117/157207409-7b93b833-50c0-4e49-a514-65134bf706ce.png)

**Comment by chr-ge** [comment]
@georgebrianb You can do this:
```jsx
<Flex align="center">
  <Divider />
  <Text padding="2">OR</Text>
  <Divider />
</Flex>
```

**Reply by Khalifa1997** [reply]
Thanks!
